core/tests/unit/ldml/keyboards/k_100_fail.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+from https://github.com/unicode-org/cldr/blob/keyboard-preview/docs/ldml/tr35-keyboards.md#element-transform
+
+-->
+<!DOCTYPE keyboard SYSTEM "../../../../../resources/standards-data/ldml-keyboards/techpreview/ldmlKeyboard.dtd">
+<keyboard locale="en" conformsTo="techpreview">
+  <info author="srl295" indicator="🙀" layout="qwerty" normalization="NFC" />
+
+  <names>
+    <name value="TestKbd" />
+  </names>
+
+  <keys>
+    <key id="gap" gap="true" />
+    <key id="a" to="a" />
+  </keys>
+
+  <layerMaps form="hardware">
+    <layerMap id="base">
+      <row keys="a gap"/>
+    </layerMap>
+  </layerMaps>
+</keyboard>
+
+<!--
+  file:///Users/srl295/src/keymanapp/keyman/common/web/types/build/src/kvk/kvk-file-writer.js:62
+        str.len = value.length + 1;
```

Comment:
* `gap` not yet supported (would be coming with touch layouts)

Reply:
It's actually "key without to="

Reply:
now supported